app/common/prescription/views/prescription-arv.html

```diff
@@ -76,7 +76,7 @@ <h3 class="panel-title">{{'CLINIC_PRESCRIPTION_CURRENT_REGIMEN'| translate}}</h3
 
             <!-- ARV Plan elements -->
             <div class="form-group form-group-lg"
-                         ng-class="{'has-error': prescriptionForm.artPlan.$invalid && vm.showMessages}">
+                         ng-class="{'has-error': prescriptionForm.artPlan.$invalid && vm.showMessages}" ng-init="vm.initArvPlans()">
```

Comment:
Great!.  I think later we can find time and do this refactory. For now please merge my PR